packages/e2e-tests/specs/block-hierarchy-navigation.test.js

```diff
@@ -96,6 +96,7 @@ describe( 'Navigating the block hierarchy', () => {
 
 		// Return to first block.
 		await openBlockNavigator();
+		await page.waitForSelector( '.editor-block-navigation__container' );
```

Comment:
I think I have PR where I included the same logic inside `openBlockNavigator`, I can refactor once I work on merging my own PR :)